src/graph/tuning.cc

```diff
@@ -379,10 +379,12 @@ static const double perChMaxTreeBws[][3] = {
   /* Blackwell (N1/N2/N4) */ {2*38.7, 2*41.4, 2*36.0},
 };
 
-NCCL_PARAM(PatEnable, "PAT_ENABLE", 2);
+NCCL_PARAM(PatEnable, "PAT_ENABLE", 0);
 static int ncclPatEnable(struct ncclComm* comm) {
   int patEnable = ncclParamPatEnable();
+#if !defined(__HIP_PLATFORM_AMD__) && !defined(__HIPCC__)
   if (comm->minCompCap < 60) return 0; // Need SM60 or higher for CUDA atomics
+#endif
   if (patEnable != 2) return patEnable;
   if (comm->nNodes != comm->nRanks) return 0; // PAT only supports 1 GPU per node
```

Comment:
Looks like setting it to 1 will force-enable it. 
If PAT is only supported for 1 GPU per node, we should not force-enable it in that scenario. 
Pls modify this part accordingly.